Graphics_Engine/Modify/CenterGradientAsymmetric.cs

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
Not sure what has happened here on git? File on my machine on master does not have a `}` at the top of the file. Probably fine though?